mempool/bench_test.go

```diff
@@ -50,3 +98,34 @@ func BenchmarkCacheRemoveTime(b *testing.B) {
 		cache.Remove(txs[i])
 	}
 }
+
+// BenchmarkClistMempoolGetNextTxBytes-8   	       2	 793475198 ns/op
+func BenchmarkClistMempoolGetNextTxBytes(b *testing.B) {
+	benchmarkMempoolGetNextTxBytes(b, enumclistmempool)
+}
+
+// BenchmarkLLRBMempoolGetNextTxBytes-8   	     322	   3565640 ns/op
+func BenchmarkLLRBMempoolGetNextTxBytes(b *testing.B) {
+	benchmarkMempoolGetNextTxBytes(b, enumllrbmempool)
+}
+
+func benchmarkMempoolGetNextTxBytes(b *testing.B, enum mpEnum) {
+	app := kvstore.NewApplication()
```

Comment:
wait. this legacy app doesn't even know about priorities. does it actually work?

Reply:
Seems true, I didn't notice that priority parse was implemented in #43